deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/ErrorCodeDescription.java

```diff
@@ -94,79 +200,42 @@ public static ErrorCodeDescription Parse(int errorCode)
                 return ArgumentNull;
             case(400006):
                 return IotHubFormatError;
-            case(400007):
-                return DeviceStorageEntitySerializationError;
-            case(400008):
-                return BlobContainerValidationError;
-            case(400009):
-                return ImportWarningExistsError;
-            case(400010):
-                return InvalidSchemaVersion;
             case(400011):
                 return DeviceDefinedMultipleTimes;
-            case(400012):
-                return DeserializationError;
             case(400013):
                 return BulkRegistryOperationFailure;
-            case (400027):
-                return ConnectionForcefullyClosedOnNewConnection;
-            case(400301):
-                return CannotRegisterModuleToModule;
-            case(401001):
-                return IotHubNotFound;
             case(401002):
                 return IotHubUnauthorizedAccess;
-            case(401003):
-                return IotHubUnauthorized;
```

Comment:
does service return 401002 for all unauth cases?

Reply:
Yes, that was our finding from the .NET side of this work